packages/frontend/component/src/components/auth-components/onboarding-page.tsx

```diff
@@ -104,6 +122,17 @@ export const OnboardingPage = ({
     () => questions?.[questionIdx],
     [questionIdx, questions]
   );
+  const isMacosDesktop = environment.isDesktop && environment.isMacOs;
+  const isWindowsDesktop = environment.isDesktop && environment.isWindows;
+
+  useEffect(() => {
+    if (environment.isDesktop) {
+      // to hide macOS window control buttons
+      apis?.ui.handleSidebarVisibilityChange(false).catch(err => {
+        console.error(err);
+      });
```

Comment:
Could we recovery after onboarding is completed?

Reply:
This window action button will appear whenever the sidebar is displayed.